src/Network/Minio/Data.hs

```diff
@@ -242,6 +242,33 @@ data ObjectInfo = ObjectInfo {
   , oiSize :: Int64
   } deriving (Show, Eq)
 
+data GetObjectOptions = GetObjectOptions {
+    -- | [ByteRangeFromTo 0 9] means first ten bytes of the source object.
+    gooRange :: Maybe Network.HTTP.Types.ByteRanges
```

Comment:
Two points here:

1. The type of `gooRange` need not be explicitly qualified as `Maybe Network.HTTP.Types.ByteRanges` - it should just be `Maybe ByteRanges` since we are importing it into the current namespace on line 32.
2. The appropriate type for `gooRange` seems to be `Maybe ByteRange` rather than `Maybe ByteRanges` since `type ByteRanges = [ByteRange] ` according to https://www.stackage.org/haddock/lts-9.2/http-types-0.9.1/Network-HTTP-Types.html#g:7 - S3 only supports a single range specification and not a list of ranges (one of the examples in [the docs](https://docs.aws.amazon.com/AmazonS3/latest/API/RESTObjectGET.html) notes explicitly that "Amazon S3 doesn't support retrieving multiple ranges of data per GET request").